Source/DfBAdminToolkit/Services/GitHubService.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using RestSharp;
+using Newtonsoft.Json;
+
+namespace DfBAdminToolkit.Services
+{
+    public class GitHubService
+    {
+
+        private readonly RestClient _client;
+
+        public GitHubService()
+        {
+            
+            _client = new RestClient(@"https://api.github.com/");
+        }
+
+        public string LatestVersion()
```

Comment:
It might make more sense to return a System.Version object type here. And then if we don't get a version back from the API maybe I can just instantiate a Version object with 0.0.0 so that the app doesn't prompt the user to update?